sdk/src/main/java/com/horizen/account/api/rpc/types/TxPoolTransaction.java

```diff
@@ -0,0 +1,85 @@
+package com.horizen.account.api.rpc.types;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonView;
+import com.horizen.serialization.Views;
+import org.web3j.utils.Numeric;
+
+import java.math.BigInteger;
+
+@JsonView(Views.Default.class)
+@JsonInclude()
+public class TxPoolTransaction {
+
+    private final static String DEFAULT_BLOCK_HASH = "0x0000000000000000000000000000000000000000000000000000000000000000";
+    private final String blockHash;         // fixed default value
```

Comment:
as was mentioned in other PR, let's make these field `public` and remove all get methods like
```
public String getBlockHash() {
        return blockHash;
    }
```
it is enough for the proper json serialization

Reply:
modifiers switch to `public` and getter methods removed